waspc/src/Wasp/Generator/Common.hs

```diff
@@ -24,13 +24,13 @@ nodeVersionRange :: SV.Range
 nodeVersionRange = SV.Range [SV.backwardsCompatibleWith latestNodeLTSVersion]
 
 latestNodeLTSVersion :: SV.Version
-latestNodeLTSVersion = SV.Version 16 0 0
+latestNodeLTSVersion = SV.Version 18 12 0
```

Comment:
Instead of just going with `18 0 0` I used the actual minor, since prior minors were not LTS. Same with npm.